### DISCUSSION TITLE
Workout Picture

### DISCUSSION THREAD

**Discussion by mbillard11** [cyberjunky/python-garminconnect#200] [Q&A]
Hey folks. Does this wrapper give you access to a workouts picture displayed on your home screen (see screenshot)? Or does it just give access to workout data? I'm looking to export that image specifically. I've been screenshotting and cropping it for my own uses. Would love a more convenient way to access it.
![Screenshot_20240406-161855](https://github.com/cyberjunky/python-garminconnect/assets/22525770/8c2a9157-68f7-4b56-8351-cffa5641d3fa)

**Comment by djamelz** [comment]
Hello, it's not a picture but a ui component which is generated with the data from the api call. You have to create it with the result of the `getactivities*` methods.
